pkg/collector/container_accelerator_collector.go

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
as no GPU for MacOS, build tag with GPU is needed.
in fact build tag is needed for all `NVIDIA` import code.

```
//go:build gpu
// +build gpu
```

Reply:
we can't force GPU and GPU related package been installed on host instance.

Reply:
ref to https://github.com/sustainable-computing-io/kepler/issues/184#issuecomment-1249013531